### PR TITLE
kill player health bar on logout

### PR DESCRIPTION
**Pull request by gabrielwr**

### CODE REVIEW

**Comment by gabrielwr** [comment]
closes #121

File: app/states/fantasyState.js [hunk 1]
```diff
@@ -57,8 +57,8 @@ const fantasyState = {
     itemGraveyard = []
 
     playerObject.movePlayer()
-    itemCollision(playerObject, itemGraveyard)
-    enemyCollision(playerObject, graveyard)
+    itemCollision(playerObject, localState.loot)
```
Comment (abc516) [inline app/states/fantasyState.js]: @gabrielwr I fixed this in previous PR, don't need localState.loot anymore, need itemGraveyard in second arg

File: app/states/fantasyState.js [hunk 2]
```diff
@@ -57,8 +57,8 @@ const fantasyState = {
...
+    enemyCollision(playerObject, graveyard, localState.loot)
```
Comment (abc516) [inline app/states/fantasyState.js]: @gabrielwr don't need third argument anymore (localState.loot)

File: app/states/spaceState.js [hunk 3]
```diff
@@ -57,8 +57,8 @@ const spaceState = {
     itemGraveyard = []
 
     playerObject.movePlayer()
-    itemCollision(playerObject, itemGraveyard)
-    enemyCollision(playerObject, graveyard)
+    itemCollision(playerObject, localState.loot)
```
Comment (abc516) [inline app/states/spaceState.js]: @gabrielwr see comments in fantasyState.js

**Review by abc516 — CHANGES_REQUESTED** [review]
Please make the changes indicated in comments.

**Pull request merged**